src/Styleguide/Pages/Artist/Routes/CV/CVQuery.ts

```diff
@@ -0,0 +1,26 @@
+import { graphql } from "react-relay"
+
+export const CVQuery = graphql`
+  query CVQuery(
+    $artistID: String!
+    $first: Int!
+    $sort: PartnerShowSorts
+    $at_a_fair: Boolean
+    $solo_show: Boolean
+    $is_reference: Boolean
+    $visible_to_public: Boolean
+  ) {
+    artist(id: $artistID) {
+      ...CVPaginationContainer_artist
+        @arguments(
+          sort: $sort
+
+          first: $first
+          at_a_fair: $at_a_fair
+          solo_show: $solo_show
+          is_reference: $is_reference
+          visible_to_public: $visible_to_public
+        )
+    }
+  }
+`
```

Comment:
Can this just move into the `CVQueryRenderer.tsx` file?

Reply:
The way its currently structured is we render the top-level CV item via a query on the router, and then render three more sub-sections via QueryRenderer. If we can combine all of these sub CV items into one then we can remove the QueryRenderer completely. (No need to use them.) The reason this query was broken into its own file is because it's shared in two locations.

Reply:
Gotcha 👍